image/templates/helm/stackrox-central/templates/01-central-00-serviceaccount.yaml

```diff
@@ -8,9 +8,11 @@ metadata:
   labels:
     {{- include "srox.labels" (list . "serviceaccount" "central") | nindent 4 }}
   annotations:
-    {{- include "srox.annotations" (list . "serviceaccount" "central") | nindent 4 }}
+    {{- if and (eq ._rox.env.openshift 4) (not ._rox.env.managedServices) }}
     serviceaccounts.openshift.io/oauth-redirectreference.main: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"central"}}'
     serviceaccounts.openshift.io/oauth-redirecturi.main: "sso/providers/openshift/callback"
+    {{- end }}
+    {{- include "srox.annotations" (list . "serviceaccount" "central") | nindent 4 }}
```

Comment:
Is there a particular reason you moved this to the bottom of the annotations list?

Reply:
No reason at all, reverted it.